### PR TITLE
Add event page

### PR DESCRIPTION
**Pull request by CahidArda**
With this PR, I am adding the event page:

image

Additionaly, there is an update to the docker-compose file fixing the proxy error in the development environment.

### CODE REVIEW

File: frontend/src/pages/EventPage/EventPage.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  }, [id, token])
+
+  const {error, isLoaded, event} = state
+
+  if (error) {
+    return <div>Error: {error.message}</div>
+  } else if (!isLoaded) {
+    return <div>Loading...</div>
+  } else {
+    console.log(event)
```
Comment (erimerkin) [inline frontend/src/pages/EventPage/EventPage.js:45]: Can we remove this part if there is no need for it?

**Review by erimerkin — APPROVED** [review]
Thanks for your additions @CahidArda. I have tested the updated proxy, it works in my local. In the future its better if we set these with an environment variable rather than hardcoding. Also added event page looks good.

File: frontend/src/pages/EventPage/EventPage.js (hunk 1 above)
Comment (CahidArda) [reply frontend/src/pages/EventPage/EventPage.js]: I have removed the console log lines from art item and event pages.

**Comment by erimerkin** [comment]
Since the commented issues are fixed, I am merging this branch to develop.

**Pull request merged**